scripts/pdf/render.py

```diff
@@ -15,108 +15,111 @@
 
 def main(loc, colorscheme):
 
-    oslist = []
-    allmd = []
-    group = []
-    ap = []
+  oslist = []
+  allmd = []
+  group = []
+  ap = []
 
-    #Checking correctness of path
-    if not os.path.isdir(loc):
-        print("Invalid directory. Please try again!", file = sys.stderr)
-        sys.exit(1)
+  # Checking correctness of path
+  if not os.path.isdir(loc):
+    print("Invalid directory. Please try again!", file = sys.stderr)
+    sys.exit(1)
 
-    #Writing names of all directories inside 'pages' to a list
-    for os_dir in os.listdir(loc):
-        oslist.append(os_dir)
+  # Writing names of all directories inside 'pages' to a list
+  for os_dir in os.listdir(loc):
+    oslist.append(os_dir)
 
-    oslist.sort()
+  oslist.sort()
 
-    #Required strings to create intermediate HTML files
-    header = "<html><head><link rel=stylesheet type=text/css href=" + colorscheme + ".css></head><body>\n"
-    footer = "</body></html>"
-    title_content = "<h1 class=titlemain>tldr pages</h1><h4 class=titlesub>Simplified and community driven man pages</h4></body></html>"
+  # Required strings to create intermediate HTML files
+  header = '<!DOCTYPE html><html><head><link rel="stylesheet" href="basic.css">'
```

Comment:
Wait, so we weren't using the basic.css at all ? But the file seems to be already there ?

Reply:
We were, but it was treated as a color scheme. So every scheme was included completely, as a file for itself. That way we had a lot of duplicate styles in 3 files. Another solution as I said in the PR description is to keep it as one file, but to remove our color schemes as they're really bad-looking.

Reply:
Nope, this looks fine. I will review when I have some time.